frontend/covid-dashboard-frontend/src/components/HomePage.js

```diff
@@ -37,155 +35,185 @@ const HomePage = ({cities, states, countries, loading, cityDetail, stateDetail,
   const [testingRateOptions, setTestingRateOptions] = useState({});
   const [isReady, setIsReady] = useState(false);
 
+  const chartContainerHeight = '375px';
 
+  const CITY_LOCATION = 'city';
+  const STATE_LOCATION = 'state';
+  const COUNTRY_LOCATION = 'country';
```

Comment:
I'm kinda a string literal hawk so I had to replace with constants to avoid generating typos in strings and crashing the app